spec/model-spec.coffee

```diff
@@ -922,6 +922,45 @@ describe 'Model', ->
       for key in updateBlacklist
         expect(json[key]).toBeUndefined()
 
+    context 'useWhitelist', ->
+      attributeKeys = expectedBlacklist = whitelist = null
+
+      beforeEach ->
+        model.useWhitelist = true
+        attributeKeys = ['possums', 'racoons', 'potatoes', 'label', 'name']
+        expectedBlacklist = ['possums', 'racoons', 'potatoes']
+        whitelist = ['label', 'name']
+
+      context "create", ->
+        it "sets the blacklist to the model's attributes except for those in the whitelist", ->
+          model.set('create_whitelist', whitelist)
+
+          for key in attributeKeys
+            model.set(key, true)
```

Comment:
It's weird to keep it as attributes... 

I wonder if makes more sense to parse this out and store it elsewhere?